src/test/javascript/spec/component/faq/faq-update.component.spec.ts

```
@@ -139,8 +165,34 @@ describe('FaqUpdateComponent', () => {
         faqUpdateComponent.save();
         tick();
         faqUpdateComponentFixture.detectChanges();
+        expect(updateSpy).toHaveBeenCalledExactlyOnceWith(courseId, { id: 6, questionTitle: 'test1Updated', faqState: 'ACCEPTED' });
+    }));
+
+    it('should propose to edit a faq', fakeAsync(() => {
+        activatedRoute.parent!.data = of({ course: { id: 1 }, faq: { id: 6 } });
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)